### DISCUSSION TITLE
Can not init web worker in demo project

### DISCUSSION THREAD

**Discussion by invm** [CodinGame/monaco-vscode-api#496] [Q&A]
Hi there,
I encounter an issue when working with the demo, when I run from the `monaco-vscode-api/demo` folder, everything seems ok and runs as planned, but when I extract the demo folder from the repo, and change all the file links in `package.json` to latest version `^8.0.4` I'm getting an error in the console:

<img width="740" alt="image" src="https://github.com/user-attachments/assets/26d9f87d-6362-46dc-ba49-03706d4a7752">

The only place where I've seen `diffStateStacksRefEq` is in the patches, which I assume are ran when building inside the repo, but as far as I know they are ran also before publish, which I expect to not have any difference between `file:../` schema and directly downloading from npm. I'm running a `vscode-server` if it is relevant.

Would appreciate any help. Thank you.

**Comment by CGNonofr** [comment]
Hello,

Did you ensure you don't have multiple versions of some packages? (you can check using `npm list vscode`)

If everything seems alright, can you provide a small reproduction repo?
